### PR TITLE
Auto-generate Trace v2

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

**Comment by dhermes** [comment]
Anything in particular you'd like reviewed?

**Comment by liyanhui1228** [comment]
Thanks for generating this! Why removing the V1 related code? We probably need to keep both V1 and V2 library as the V1 API is still serving.

**Comment by lukesneeringer** [comment]
Whups, sorry about that. Re-added the v1 guts.

Note that users explicitly wanting to use v1 will have to `from google.cloud import trace_v1`; just importing `trace` will get them v2.

File: trace/google/cloud/trace_v2/proto/tracing_pb2.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+from google.protobuf import descriptor_pb2
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
+from google.devtools.cloudtrace_v2.proto import trace_pb2 as google_dot_devtools_dot_cloudtrace__v2_dot_proto_dot_trace__pb2
```
Comment (liyanhui1228) [inline trace/google/cloud/trace_v2/proto/tracing_pb2.py]: Importing from `google.devtools` is causing the tests failure, can we rename it?

**Comment by liyanhui1228** [comment]
CircleCI is blocked by error-reporting system tests. Verified locally that the trace tests passed. LGTM to merge.

**Pull request merged**